internal/dsync/drwmutex.go

```diff
@@ -247,6 +253,26 @@ func (dm *DRWMutex) startContinousLockRefresh(lockLossCallback func(), id, sourc
 	}()
 }
 
+func forceUnlock(ctx context.Context, ds *Dsync, id string) {
+	restClnts, _ := ds.GetLockers()
+
+	var wg sync.WaitGroup
+	for index, c := range restClnts {
+		wg.Add(1)
+		// Send refresh request to all nodes
+		go func(index int, c NetLocker) {
+			defer wg.Done()
+			args := LockArgs{
+				UID: id,
+			}
+			ctx, cancel := context.WithTimeout(ctx, drwMutexForceUnlockCallTimeout)
```

Comment:
Tend to agree with Harsha, just move this outside and use the same ctx for all. No need to waste resources, and it doesn't make the code uglier.

Reply:
> Tend to agree with Harsha, just move this outside and use the same ctx for all. No need to waste resources, and it doesn't make the code uglier.

In fact, ctx here equals context.Background(), I can replace it with context.Background() to make it easier which makes sense as well